v4/source/posts.yaml

```diff
@@ -397,6 +397,33 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/channels/{channel_id}/posts/unread':
```

Comment:
Can we include the user ID in the path? That way a bot or an admin could get the unread posts for a different user if they needed to

Reply:
Would we want that? Since this is a route designed specifically for one use case in our apps, it's doesn't seem as likely that other people will be using it. 0/5 though

Reply:
My counter would be "why not?" I don't see a downside to adding the user ID to the path (if you want it for you you can use "me" instead of your user ID) and it gives more power to admins using the API. Also that would be consistent with other very similar endpoints we already have https://github.com/mattermost/mattermost-api-reference/blob/master/v4/source/channels.yaml#L1340

Reply:
I wasn't aware of the existing endpoint of that style. I think it makes sense even if I can't think of endpoints for it at this very moment